### ISSUE TITLE
Telegram: топик не обновляется на лету и событие смены топика не отображается в журнале

### ISSUE THREAD

**Issue by dartraiden** [miranda-ng/miranda-ng#4673]
Меняем топик в чате.

Проблема 1: топик остаётся в базе старым до переподключения

Проблема 2: из-за этого в журнале нет события о смене топика. Невозможно вообще никак узнать, что топик поменялся, поскольку он даже нигде не фигурирует, кроме как в базе.

Официальный клиент показывает смену топика исправно, значит, это событие как-то приходит в клиент.

**Comment by georgehazan** [comment]
> Официальный клиент показывает смену топика исправно, значит, это событие как-то приходит в клиент.

Хрен там ночевал в официальном клиенте, просто при заходе на страницу опций групчата он всегда в явном виде заново сдергивает с сервера всю инфу. Хорошо видно, что в миранду обновление инфы с сервера приходит только при смене названия или аватара групчата

**Comment by dartraiden** [comment]
Да, я спутал. Официальный клиент в журнале показывает смену названия чата, а не топика.
